.github/workflows/add-to-project.yaml

```diff
@@ -1,4 +1,4 @@
-name: sync-board
+name: add-to-project
 
```

Comment:
@akeller you had a better name all along 👇

Reply:
Ayyyye! 💃